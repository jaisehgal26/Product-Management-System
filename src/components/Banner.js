import React from "react";
import Fade from "react-reveal/Fade";
import "../App.css";

export default function Banner() {
  return (
    <div className="container-xxl bg-primary my-5 py-5 pt-0">
      <Fade bottom>
        <div className="container">
          <div className="row g-4 align-items-center">
            <div className="col-lg-6">
              <h1 className="display-4 text-light mb-0 Playfair">
                The Best Bakery In Your City
              </h1>
            </div>
            <div className="text-light col-lg-6 text-lg-end">
              <div className="d-inline-flex align-items-center text-start">
                <i className="fa fa-phone-alt fa-4x flex-shrink-0"></i>
                <div className="ms-4">
                  <p className="fs-5 fw-bold mb-0 ">Call Us</p>
                  <p className="fs-1 fw-bold mb-0 ">+012 345 6789</p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </Fade>
    </div>
  );
}
