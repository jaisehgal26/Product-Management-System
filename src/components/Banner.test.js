import Banner from "./Banner";
import { shallow } from "enzyme";
import * as React from "react";
import toJson from "enzyme-to-json";

test("renders Banner caption", () => {
  const wrapper = shallow(<Banner />);
  expect(wrapper.find("h1").text()).toContain("The Best Bakery In Your City");
});
test("includes two paragraphs", () => {
  const wrapper = shallow(<Banner />);
  expect(wrapper.find("p").length).toEqual(2);
});
test("should render correctly", () => {
  const wrapper = toJson(shallow(<Banner />));
  expect(wrapper).toMatchSnapshot();
});
