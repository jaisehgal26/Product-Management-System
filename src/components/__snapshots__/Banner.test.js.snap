// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div
  className="container-xxl bg-primary my-5 py-5 pt-0"
>
  <Fade
    bottom={true}
  >
    <div
      className="container"
    >
      <div
        className="row g-4 align-items-center"
      >
        <div
          className="col-lg-6"
        >
          <h1
            className="display-4 text-light mb-0 Playfair"
          >
            The Best Bakery In Your City
          </h1>
        </div>
        <div
          className="text-light col-lg-6 text-lg-end"
        >
          <div
            className="d-inline-flex align-items-center text-start"
          >
            <i
              className="fa fa-phone-alt fa-4x flex-shrink-0"
            />
            <div
              className="ms-4"
            >
              <p
                className="fs-5 fw-bold mb-0 "
              >
                Call Us
              </p>
              <p
                className="fs-1 fw-bold mb-0 "
              >
                +012 345 6789
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </Fade>
</div>
`;
